app/views/containers/show.html.erb

```diff
@@ -108,7 +107,9 @@
         </div>
       </div>
       <div class="tab-pane" id="logs">
-        <pre class="terminal terminal-output pre-scrollable"><code><%= Docker::Container.get(@container.uuid).logs(:stdout => true, :tail => 100) %></code></pre>
+        <pre class="terminal terminal-output pre-scrollable"><code>
+          <%= logs(@container.uuid, :stdout => true, :tail => 100) %>
```

Comment:
`logs` is called here using the uuid of the container. However the method expects the Container object. 

It results on `undefined method compute_resource for #<String:0x007f2e9aa83980>` on Container#show on a container that is on.